entc/gen/template/dialect/sql/query.tmpl

```diff
@@ -7,7 +7,9 @@ in the LICENSE file in the root directory of this source tree.
 {{/* Additional fields for the builder. */}}
 {{ define "dialect/sql/query/fields" }}
 	{{- with $.ForeignKeys }}
-		withFKs bool
+		{{- if not ($.FeatureEnabled "exposefks") }}
+			withFKs bool
```

Comment:
This is always true if the FF is enabled, so don't generated it if the FF is on.